packages/replayio/src/utils/initialization/getPackageManagerCommand.ts

```diff
@@ -0,0 +1,22 @@
+import { basename, sep } from "path";
+import { isPackageManagerInstalled } from "./isPackageManagerInstalled";
+import { PackageManager } from "./types";
+
+export function getPackageManagerCommand(): PackageManager | undefined {
+  // If this is a global install, it's possible we can detect the package manager that way
+  const path = basename(__filename);
+  if (path.includes("pnpm" + sep)) {
+    return "pnpm";
+  } else if (path.includes("yarn" + sep)) {
+    return "yarn";
+  }
+
+  // Otherwise let's see which package manager(s) are installed
```

Comment:
I have all of them installed. It would be better to lookup the closest lockfile

Reply:
Hm. I dunno. I think the likelihood of you running the CLI from within a project folder that also happens to be the way you installed `replayio` seems low. I either run it from my home folder (default terminal location) or I run it from whatever project I'm working in (which may use Yarn or PNPM, but that's not how I globally installed the CLI)